### PR TITLE
refactor: use fastify-tsconfig in ts template (#273)

### PR DESCRIPTION
**Pull request by alemagio**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by zekth — APPROVED** [review]
LGTM

**Review by mcollina — APPROVED** [review]
lgtm

File: generate.js [hunk 1]
```diff
@@ -59,6 +59,7 @@ const typescriptTemplate = {
     '@types/node': cliPkg.devDependencies['@types/node'],
     '@types/tap': cliPkg.devDependencies['@types/tap'],
     concurrently: cliPkg.devDependencies.concurrently,
+    'fastify-tsconfig': cliPkg.devDependencies['fastify-tsconfig'],
```
Comment (mcollina) [inline generate.js:62]: I think we are missing this in package.json.

**Review by mcollina — CHANGES_REQUESTED** [review]

File: generate.js (hunk 1 above)
Comment (alemagio) [reply generate.js:62]: We added it when we added the `generate-plugin` command.

**Review by mcollina — APPROVED** [review]
lgtm

I missed this!

**Pull request merged**